game/scripts/npc/abilities/techies_land_mines.txt

```diff
@@ -61,12 +61,12 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "building_damage_pct"                             "30"
+        "building_damage_pct"                             "125" //oaa
```

Comment:
I've been told in past this is dmg to bosses but in testing don't know if that how it works. Eitherway, the thought process behind it is simple: Mines should absolutely do damage to bosses (100%) and it's 125% since the mines don't receive any spell amp.

Reply:
I modified only some abilities that deal bonus dmg to buildings to also deal bonus dmg to bosses. If an ability has a penalty for building dmg, I didnt include that penalty. So techies mines already do full dmg to bosses. This literally doesn't change anything and techies mines don't need to do bonus dmg to bosses because bosses have -50% magic resistance.

Reply:
> I modified only some abilities that deal bonus dmg to buildings to also deal bonus dmg to bosses. If an ability has a penalty for building dmg, I didnt include that penalty. So techies mines already do full dmg to bosses. This literally doesn't change anything and techies mines don't need to do bonus dmg to bosses because bosses have -50% magic resistance.

kk than I'll revert this.